<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameToolController">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="toolController">
        <property name="toolTip">
         <string>The tool and its settings to be used for this operation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelToolController">
        <property name="text">
         <string>Tool Controller</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutputFile">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="SetOutputFileName">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="OutputFileName">
        <property name="toolTip">
         <string>Enter the filename where the probe points should be written</string>
        </property>
        <property name="text">
         <string>ProbePoints.txt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelDistance">
        <property name="text">
         <string>Probing Distance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="sbDistance"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbGeneration">
     <property name="title">
      <string>Auto Generate</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="2">
       <widget class="QSpinBox" name="sbCountGen">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pbReplaceGen">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Replace All</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelNumberPoints">
        <property name="text">
         <string>Number of points</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pbAddGen">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cbReferenceGen"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelReference">
        <property name="text">
         <string>Reference</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formLoc" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QWidget" name="addRemoveEdit" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QPushButton" name="addLocation">
           <property name="toolTip">
            <string>Opens a dialog to add arbitrary locations.</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="removeLocation">
           <property name="toolTip">
            <string>Remove selected location from the list. The operation is no longer applied to them.</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="stopAddition">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTableWidget" name="baseList">
        <property name="toolTip">
         <string>List of locations to be processed.</string>
        </property>
        <column>
         <property name="text">
          <string>Reference</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>X</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Y</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Z</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
